lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -117,8 +135,16 @@ def _proc_upload_reads_params(self, params):
         if not shock and revid:
             raise ValueError('Cannot specify a reverse reads file in shock ' +
                              'with a local forward reads file')
```

Comment:
The code should check that <= 1 of the rev* sources are specified, and that if it is specified, it matches the fwd* source.